corehq/apps/reports/standard/monitoring.py

```diff
@@ -1355,7 +1355,7 @@ def _group_cell(self, group_id, group_name):
         url = self._make_url(base_url, params)
 
         return util.format_datatables_data(
-            self._html_anchor_tag(url, group_name),
+            self._html_anchor_tag(url, group_name.encode('utf-8')),
```

Comment:
Interesting that this returns bytes and not Unicode. I would have expected the fix to be to make the template string at [line 1274](https://github.com/dimagi/commcare-hq/blob/fb9da0ca07e9e834511cf53b9e27a27ebba24605/corehq/apps/reports/standard/monitoring.py#L1274) a Unicode literal (prefixed with a `u`) instead of a byte string.